framework/src/transfers/MultiAppCloneReporterTransfer.C

```diff
@@ -54,12 +53,24 @@ MultiAppCloneReporterTransfer::MultiAppCloneReporterTransfer(const InputParamete
                                                     _from_reporter_names)
                            : getReporterNamesHelper(_name, _to_obj_name, _from_reporter_names))
 {
+  // Deprecating direction proves fatal for this parameter for this class
+  if (isParamValid("multi_app"))
+    paramError("The multi_app parameter is no longer valid for this class, use to_multi_app");
```

Comment:
Did you mean "`from_multi_app`" instead of  `to_multi_app`?

Reply:
changed